akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -835,6 +835,7 @@ private[remote] abstract class ArteryTransport(_system: ExtendedActorSystem, _pr
             case OptionVal.Some(snd) =>
               snd.tell(ActorSystemTerminatingAck(localAddress), ActorRef.noSender)
               if (inControlStream)
+                // FIXME, why does this side timeout? It received the ActorSystemTerminating, this system isn't terminating
```

Comment:
anything we need to look into?

Reply:
i added that when debugging the non-cluster case, removing

Reply:
still to be removed?